### PR TITLE
feat: validate composite before include it in the bundle

### PR DESCRIPTION
**Pull request by leanmendoza**
# What?
- Support CommonJS build for `@dcl/ecs`
- Include `@dcl/ecs` in `sdk-commands`
- At build-time, only composite that can be instanced are added to composite provider
- Double-check with adding a try-catch while instancing the main.composite

### CODE REVIEW

File: packages/@dcl/ecs/tsconfig.cjs.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "CommonJS",
+    "outDir": "dist-cjs"
```
Comment (gonpombo8) [inline packages/@dcl/ecs/tsconfig.cjs.json:5]: maybe `dist/cjs` ?

Comment (leanmendoza) [reply packages/@dcl/ecs/tsconfig.cjs.json:5]: I'll try it, I guessed there could be some problems, but I didn't tried

Comment (menduz) [reply packages/@dcl/ecs/tsconfig.cjs.json:5]: it is better to not mix two module systems inside `dist`, i'd keep dist-cjs

File: packages/@dcl/sdk-commands/src/logic/bundle.ts [hunk 2]
```diff
@@ -203,6 +204,12 @@ function compositeLoader(components: BundleComponents, options: CompileOptions):
             )
             contents = `export const compositeFromLoader = {${data.compositeLines.join(',')}}`
             watchFiles = data.watchFiles
+
+            if (data.withErrors) {
+              components.logger.log(
```
Comment (menduz) [inline packages/@dcl/sdk-commands/src/logic/bundle.ts:209]: we have a visually impacting "beautiful logger" for errors if you'd like to use it

File: packages/@dcl/sdk-commands/src/logic/composite.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    const engine = Engine()
+    try {
+      const composite = compositeProvider.getCompositeOrNull(compositeSource)!
+      Composite.instance(engine, composite, compositeProvider)
+      compositeLines.push(`'${composite.src}':${JSON.stringify(Composite.toJson(composite.composite))}`)
+    } catch (err: any) {
+      components.logger.log(`Composite '${compositeSource}' can't be instanced ${err.stack}`)
+      withErrors = true
+    }
+  }
```
Comment (menduz) [inline packages/@dcl/sdk-commands/src/logic/composite.ts:51]:
```suggestion
  const composites: Record<string, string> = {}

  for (const compositeSource in composites) {
    const engine = Engine()
    try {
      const composite = compositeProvider.getCompositeOrNull(compositeSource)!
      Composite.instance(engine, composite, compositeProvider)
      composites[composite.src] = JSON.stringify(Composite.toJson(composite.composite))
    } catch (err: any) {
      components.logger.log(`Composite '${compositeSource}' can't be instanced ${err.stack}`)
      withErrors = true
    }
  }
```

then `JSON.stringify(composites)`

File: packages/@dcl/sdk-commands/src/logic/bundle.ts (hunk 2 above)
Comment (leanmendoza) [reply packages/@dcl/sdk-commands/src/logic/bundle.ts:209]: I'll do the improvement later

File: packages/@dcl/sdk-commands/src/logic/composite.ts (hunk 3 above)
Comment (leanmendoza) [reply packages/@dcl/sdk-commands/src/logic/composite.ts:51]: I defer this to another PR

**Pull request merged**